helm/cluster-gcp/values.yaml

```diff
@@ -45,12 +45,20 @@ bastion:
 controlPlane:
   apiAllowlistSubnets: ""  # A comma separated list of cidrs that will have access to the kubernetes API.
   instanceType: n2-standard-4
-  rootVolumeSizeGB: 100
-  etcdVolumeSizeGB: 100
-  containerdVolumeSizeGB: 100
-  kubeletVolumeSizeGB: 100
+  rootVolume:
+    sizeGB: 100
+    diskType: "pd-ssd"  # "pd-standard" is the default one, the other only supported type is "pd-ssd"
```

Comment:
The comment here conflicts with the value set

Reply:
Oh I meant to say that by default CAPG sets it to "pd-standard" but we want to override it to "pd-ssd".
I will rephrase it